doc/rados/configuration/mon-config-ref.rst

```diff
@@ -467,6 +467,7 @@ by setting it in the ``[mon]`` section of the configuration file.
 
 .. index:: Ceph Storage Cluster; capacity planning, Ceph Monitor; capacity planning
 
+.. _storage-capacity:
```

Comment:
please add an empty line after the label. see the examples at https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-ref . and the warning messages from sphinx is specific,

```
ceph/doc/rados/configuration/mon-config-ref.rst:471: WARNING: Explicit markup ends without a blank line; unexpected unindent.
ceph/doc/rados/troubleshooting/troubleshooting-osd.rst:245: WARNING: Explicit markup ends without a blank line; unexpected unindent.
```

Reply:
Explciit yes, elliptical arguably.  I added a blank line after the labels, awaiting rechecks.